xcm/pallet-xcm/src/lib.rs

```diff
@@ -786,10 +849,12 @@ pub mod pallet {
 				origin_location,
 				message,
 				hash,
-				max_weight, //.ref_time(),
-				max_weight, //.ref_time(),
+				max_weight,
+				max_weight,
 			);
-			let result = Ok(Some(outcome.weight_used().saturating_add(100_000_000)).into());
+			let result =
+				Ok(Some(outcome.weight_used().saturating_add(Weight::from_ref_time(100_000_000)))
+					.into());
```

Comment:
```suggestion
			let result = Ok(Some(outcome.weight_used().saturating_add(T::WeightInfo::execute())).into());
```

Reply:
Note we should not be seeing *any* hard-coded weight values *at all* with this PR.